app/uk/gov/hmrc/bindingtariffclassification/scheduler/DaysElapsedJob.scala

```diff
@@ -31,14 +31,16 @@ import scala.concurrent.duration.FiniteDuration
 @Singleton
 class DaysElapsedJob @Inject()(appConfig: AppConfig, caseService: CaseService) extends ScheduledJob {
 
+  private implicit val carrier: HeaderCarrier = HeaderCarrier()
```

Comment:
@ed0906 
This is better than the code before, but is there a way to use the default implicit `HeaderCarrier` of the microservice without instantiating it?

Reply:
The "default implicit" one is defined on the `BaseController` which pulls fields off the `request` we wont be able to use that one as it's a scheduled job and doesn't have a `request`